src/applications/vaos/tests/e2e/page-objects/ClosestCityStatePageObject.js

```diff
@@ -12,6 +12,9 @@ class ClosestCityStatePageObject extends PageObject {
     cy.url().should('include', 'closest-city');
     cy.axeCheckBestPractice();
 
```

Comment:
Not expecting assertUrl to click a button. I would move the assertion to the test.